akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -176,6 +185,10 @@ private[akka] class SystemMessageDelivery(
           case msg @ (_: SystemMessage | _: AckedDeliveryMessage) ⇒
             if (unacknowledged.size < maxBufferSize) {
               seqNo += 1
+              if (unacknowledged.isEmpty)
+                ackTimestamp = System.nanoTime()
```

Comment:
it's reset in the `ack` method above, i.e. when removed from buffer, but I think it's also needed here for a new message coming in and it was a long time since previous ack